lectures/inequality.md

```diff
@@ -417,83 +616,98 @@ varlist = ['n_wealth',   # net wealth
 df = df_income_wealth
 
 # create lists to store Gini for each inequality measure
-
-Ginis = []
+results = {}
 
 for var in varlist:
     # create lists to store Gini
-    ginis = []
-    
+    gini_yr = []
     for year in years:
         # repeat the observations according to their weights
         counts = list(round(df[df['year'] == year]['weights'] ))
         y = df[df['year'] == year][var].repeat(counts)
         y = np.asarray(y)
         
         rd.shuffle(y)    # shuffle the sequence
-        
+      
         # calculate and store Gini
         gini = qe.gini_coefficient(y)
-        ginis.append(gini)
+        gini_yr.append(gini)
         
-    Ginis.append(ginis)
+    results[var] = gini_yr
+
+# Convert to DataFrame
+results = pd.DataFrame(results, index=years)
+results.to_csv("_static/lecture_specific/inequality/usa-gini-nwealth-tincome-lincome.csv", index_label='year')
 ```
 
+While the data can be computed using the code above, we will import a pre-computed dataset from the lecture repository.
+
+<!-- TODO: update from csv to github location -->
+
 ```{code-cell} ipython3
-ginis_nw, ginis_ti, ginis_li = Ginis
+ginis = pd.read_csv("_static/lecture_specific/inequality/usa-gini-nwealth-tincome-lincome.csv", index_col='year')
+ginis.head(n=5)
 ```
 
 Let's plot the Gini coefficients for net wealth, labor income and total income.
 
+Looking at each data series we see an outlier in gini coefficient computed for 1965. 
```

Comment:
gini -> Gini

Reply:
this has been done in today's narrative updates 👍